--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -11,7 +11,10 @@
 # SPDX-License-Identifier: Apache-2.0
 # *******************************************************************************
 
-module(name = "score_tooling")
+module(name = "score_tooling",
+    version = "2.2.1",
+    compatibility_level = 2
+)
 
 ###############################################################################
 # Core Dependencies
