--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -11,7 +11,10 @@
 # SPDX-License-Identifier: Apache-2.0
 # *******************************************************************************
 
-module(name = "score_cpp_policies")
+module(name = "score_cpp_policies",
+    version = "0.0.1",
+    compatibility_level = 0
+)
 
 bazel_dep(name = "bazel_skylib", version = "1.8.2")
 bazel_dep(name = "platforms", version = "0.0.10")
